### PR TITLE
Code of Conduct Draft

### PR DESCRIPTION
**Pull request by jasdumas**
I adapted this from https://user2018.r-project.org/code_of_conduct/ I also think as the conf gets closer we should confer with the rest of the organizers about being designated responders for the coc and providing either their email/cell phone on this page for the need help section

### CODE REVIEW

File: content/codeofconduct.md [hunk 1]
```diff
@@ -3,4 +3,57 @@ title: "Code of Conduct"
...
+
+All attendees, speakers, sponsors and volunteers at Noreast'R Conference are required to agree to the following code of conduct. Organizers will enforce this code throughout and expect cooperation in ensuring a safe environment for all.
+
+We do not tolerate harassment of Noreast'R Conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, conference events, Twitter and other online media.
+
+Participants engaging in harassing behavior may be sanctioned in any way organizers deem appropriate. Participants may be given a formally warning to stop the behaviour and expelled if they continue or immediately expelled from the conference without refund and their details will be forwarded to partner institutions and events. Additionally if the harassing behaviour is part of a sponsor display or a talk the display will be removed or the talk stopped as well the other responses listed above.
```
Comment (jhollist) [inline content/codeofconduct.md]: "formal warning" not "formally warning"

**Comment by jhollist** [comment]
I think it looks good.  Found one small typo.  Can you update and push?  I'll merge after that.

File: content/codeofconduct.md (hunk 1 above)
Comment (jasdumas) [reply content/codeofconduct.md]: thanks for that catch!

**Comment by jasdumas** [comment]
ready for merging now 😄

**Pull request merged**